DESCRIPTION

```diff
@@ -19,20 +19,31 @@ URL: https://mlr3spatial.mlr-org.com,
     https://github.com/mlr-org/mlr3spatial
 BugReports: https://github.com/mlr-org/mlr3spatial/issues
 Depends:
-    R (>= 3.1.0)
+    mlr3 (>= 0.11.0),
+    R (>= 3.1.0),
+    terra (>= 1.2-10)
```

Comment:
I am getting

```
ℹ Loading mlr3spatial
Warning messages:
1: replacing previous import ‘data.table::shift’ by ‘terra::shift’ when loading ‘mlr3spatial’ 
2: replacing previous import ‘data.table::copy’ by ‘terra::copy’ when loading ‘mlr3spatial’
```

We might need to be more selective for some imports.

Reply:
Rethinking this, if we want to add support for more pkgs in the future, we should only import {terra} and not depend on it.